Sources/TuistLoader/SwiftPackageManager/PackageInfoMapper.swift

```diff
@@ -209,6 +211,17 @@ public final class PackageInfoMapper: PackageInfoMapping {
                     }
                 }
             }
+        // Include dependencies added as binary targets
+        let remoteXcframeworksPath = path.appending(components: ["artifacts", "tuist"])
```

Comment:
I don't fully follow why are we globbing in the `tuist` directory? Is that something that's constant regardless of the project?

Reply:
I'm actually not sure where this `tuist` is coming from, but there is no `tuist` mentioned in the fixture so I was assuming it was somehow constant 🤔

Reply:
I don't think it is – xcframeworks resolved by SPM are in `.build/artifacts`. I'd try removing the `tuist` path component and see if you run into any issues

Reply:
Can you direct me to the fixture where I can see that behavior? For example, `tuist install` in the `multiplatform_app_with_sdk` fixture has artifacts and they are are all directly in `.build/artifacts`. Just trying to understand this a bit better :)

Reply:
That one doesn't have a direct xcframework dependency :)
You can use the `app_with_spm_dependencies` I have edited in this PR 🙏

Reply:
Thanks for pointing me to that fixture.

Won't the location be different when the `Package.swift` is in the root which is a valid location?

I think it'd be also worth having a dedicated fixture for showcasing this feature. We're trying to move away from the monolith `app_with_spm_dependencies` as it's more difficult to work with.

Reply:
Thanks for adding the fixture @danieleformichelli!

The location in `.build/artifacts` is derived from the parent directory of the `Package.swift`. If the `Package.swift` is not in the `Tuist` directory but in the root, the artifacts are in `{path-to-project}/.build/artifacts/{root_directory}/...`. For example, when I change the structure of the fixture to this scenario, `Sentry` is in `.../app_with_spm_xcframework_dependency/.build/artifacts/app_with_spm_xcframework_dependency/Sentry`.

We'll need to remove the hardcoded path in favor of using the name of the `Package.swift`'s parent directory.